recipes/virtualenv/build.sh

```diff
@@ -0,0 +1,4 @@
+
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Please move this into the `build/script` section and remove `build.sh` and `bld.bat`.